operator/templates/cassandra-healthcheck-sh.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cassandra-healthcheck-sh
+data:
+  cassandra-healthcheck.sh: |
+    echo "Hello!"
```

Comment:
This script should be like here https://github.com/mesosphere/dcos-commons/blob/master/frameworks/cassandra/src/main/dist/svc.yml#L154-L168
But on readiness check phase ConfigMap is not available.
@mpereira  Do you have some ideas how to get around it?

Reply:
What's the evidence that it wasn't available?

Reply:
resolved